Monika After Story/game/script-topics.rpy

```diff
@@ -15985,27 +15985,34 @@ label monika_eating_meat:
     m 3eud "If you're asking if I would do so for {i}survival{/i}, I wouldn't hesitate. {w=0.2}It's not that eating meat is distressing for me or anything."
     m 7eud "I told you before, I'm vegetarian because of the impact of the mass-production of meat on the environment...{w=0.2}{nw}"
     extend 2euc "which also includes fish farming, so I'm not a pescatarian."
-    m 2rsc "...That said, I don't consider myself vegan either. {w=0.3}{nw}"
-    extend 4eud "Sure, the consumption of animal products contributes to environmental damage, but a lot of vegan alternatives have their own issues too..."
-    m 4euc "These include things like importing perishable products across great distances and mass-farming in conditions that are both cruel for workers and a strain on the local ecosystem."
+    m 2rsc "...That said, I don't consider myself vegan either."
+    m 7eua "I understand a lot of vegans are so not just for health or environmental reasons, but also ethical ones, so I think that's commendable in its own way."
```

Comment:
```suggestion
    m 7eua "I understand a lot of vegans are so not just for health or environmental reasons, but also ethical ones.{w=0.2} {nw}"
    extend 7eub "I think that's commendable in its own way."
```

Reply:
That line is controversial with one of the existing topics.
```
"Well, I guess a lot of people are more concerned about it being inhumane and all that..."
"I don't really care as much about that part."
"It's weird, like we only care about killing the things that we personally relate to as a species."
"Most people are fine with killing bugs because they're icky."
"And of course, we all kill billions of microorganisms daily without even giving it thought."
"But suddenly, if they're just a little bit bigger, it's murder!"
```

Reply:
Tru

Reply:
Well yeah, but she directly says she doesn't care so much about that part, she doesn't say anything negative about people who do care about it. I think that's a very important distinction to make. We're elaborating here. Plus in the next sentence, she reiterates her personal convictions. So in my opinion, it doesn't clash at all, but it builds upon her personality of looking at a topic from multiple angles, like she talks about in another topic I forgot the name of. In this topic, she looks at eating meat and is very diplomatic about that, even though her first topic indicates no such inclination. So I think it's not only valid for her to include a sentence like this, but also very in-character and doesn't clash with the previous topic as much as it adds to it.

Reply:
> she doesn't say anything negative about people who do care about it.

I'm not asking to add a negative reaction.

> doesn't clash with the previous topic as much as it adds to i

I do think there is a conflict, again:
The OG topic:
```
"It's weird <...>"
"But suddenly <...> it's murder!"
```
Your line:
```
I think that's commendable in its own way.
```

>  bringing up vegan food without bringing up ethics

She's already stated her position about ethics, hence I'm saying the topics are controversial.

Reply:
She's not talking about her position here, that's the next sentence. She's talking about a very uncontroversial topic, which is "some people do it for ethics, good for them" kind of deal. Is there another way I can write this sentence to bring this more to the forefront, you think? Or do you think her saying "good for them" is also controversial? I really want to include a line that at least acknowledges that there is a growing population of people who do care about the ethics side of things. Would you not agree that it's an important thing to bring up when elaborating on a philosophical position?

In the first topic, she says how most people view it right? I think it's important to acknowledge that there are other kinds of people who don't think like that. Bringing it up isn't weakening her own stance on the issue.

Reply:
> She's not talking about her position here

I don't get it, she says "I think" in your edit, so it does state *her* position.

> Or do you think her saying "good for them" is also controversial?

I think that'd sound rude.

> I really want to include a line that at least acknowledges that there is a growing population of people who do care about the ethics side of things.

She already acknowledged that and stated her position.
`Well, I guess a lot of people are more concerned about it being inhumane and all that...`

Reply:
Her position is she is concerned about the environment. Thinking reasons other do something is commendable doesn't mean you agree with the rest of their position. 

Yeah, it would sound kind of rude to put it plainly like that. Which is why I'm wondering if there's a better way to write that sentiment without sounding like she fully agrees with them?

She acknowledged that in her first topic yes, but didn't elaborate more on it than that. Also, I think it was in relation to vegetarianism, rather than veganism. "Most people" here would be referring to people who stop eating meat, right? So there is a foundation for bringing that up again with veganism, which is the ethics side of things taken to its conclusion.

Do you have a suggestion for changing this line, or do you have a suggestion for a better lead-in perhaps?

Reply:
> She acknowledged that in her first topic yes, but didn't elaborate more on it than that.

She says she finds it a weird reason, what else regarding that she could elaborate?

> in relation to vegetarianism, rather than veganism.

Why would her position differ? Two sides of the same coin.

> Do you have a suggestion for changing this line, or do you have a suggestion for a better lead-in perhaps?

Tbh, I see only about 2 edits that actually add value to the topic and make it better. The rest feels like changes for the sake of changes, or I don't see what you wanted to show with them.

Reply:
See, this is my basis for writing this topic in the first place. The first topic was written in a way that made veganism seem like "a more restrictive form of vegetarianism with nutrient deficiencies". That's a very shallow view of it, and I contributed to writing that at a time where I knew very little about it. Now that I know a lot more, I decided it was time to re-write it focusing on the reason people choose veganism over vegetarianism, but not in a pro-vegan way, rather from a slightly more educated POV of a vegetarian (I did a lot of research on vegetarian positions on veganism before writing it).

The edits only add the ethical line to the "I don't consider myself a vegan either", and a reiteration of why she's not a vegan, and moves the pros and cons down further, so I'm not sure which 2 edits you're referring to, or what other edits you don't understand. I'm just trying to understand what exactly you want me to change?